spec/coder_spec.lua

```diff
@@ -1719,6 +1719,15 @@ describe("Pallene coder /", function()
         end)
     end)
 
+
+    describe("Toplevel corner cases", function ()
```

Comment:
I think you could put these tests in the existing "Corner cases of scoping" section.